src/coreclr/vm/virtualcallstub.cpp

```diff
@@ -567,12 +638,16 @@ void VirtualCallStubManager::Init(LoaderAllocator *pLoaderAllocator)
                 DWORD cPagesRemainder = cWastedPages % 2; // We'll throw this at the cache entry heap
 
                 indcell_heap_reserve_size += cPagesPerHeap * GetOsPageSize();
+#ifdef FEATURE_VIRTUAL_STUB_DISPATCH
                 cache_entry_heap_reserve_size += (cPagesPerHeap + cPagesRemainder) * GetOsPageSize();
+#endif // FEATURE_VIRTUAL_STUB_DISPATCH
             }
 
+#ifdef FEATURE_VIRTUAL_STUB_DISPATCH
```

Comment:
Is it correct to remove this whole consistency check instead of just remove `cache_entry_heap_reserve_size` from it?